controller/roles_blackbox_test.go

```diff
@@ -0,0 +1,133 @@
+package controller_test
+
+import (
+	"github.com/fabric8-services/fabric8-auth/app"
+	"testing"
+
+	"github.com/jinzhu/gorm"
+
+	"github.com/fabric8-services/fabric8-auth/account"
+	"github.com/fabric8-services/fabric8-auth/app/test"
+	rolemodel "github.com/fabric8-services/fabric8-auth/authorization/role/model"
+	scoperepo "github.com/fabric8-services/fabric8-auth/authorization/role/scope/repository"
+	roleservice "github.com/fabric8-services/fabric8-auth/authorization/role/service"
+
+	. "github.com/fabric8-services/fabric8-auth/controller"
+	"github.com/fabric8-services/fabric8-auth/gormtestsupport"
+
+	testsupport "github.com/fabric8-services/fabric8-auth/test"
+	"github.com/goadesign/goa"
+	"github.com/satori/go.uuid"
+
+	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
+)
+
+type TestRolesRest struct {
+	gormtestsupport.DBTestSuite
+}
+
+func (s *TestRolesRest) SetupSuite() {
+	s.DBTestSuite.SetupSuite()
```

Comment:
No need to explicitly declare SetupSuite() if it only calls s.DBTestSuite.SetupSuite()

Reply:
b550a4c